ui/src/clientLibraries/graphics/GoLogo.tsx

```diff
@@ -1,6 +1,7 @@
 // Libraries
 import React, {SFC} from 'react'
 
+/* eslint-disable no-mixed-spaces-and-tabs */
```

Comment:
these should have a matching `/* eslint-enable */` at the end of the file, in the name of compositional ease (changing include style, copy pasta, etc)